client/src/pages/Registration.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
```

Comment:
I see you created the form on this branch, and the code looks good, but will you please attach a screenshot

Reply:
i did check the read Me